lib/actions/help.coffee

```diff
@@ -63,17 +82,21 @@ general = (params, options, done) ->
 			return 'primary'
 		return 'secondary'
 
-	print(parse(groupedCommands.primary))
+	print parse(groupedCommands.primary).sort(getManualSortCompareFunction(
+		manuallySortedPrimaryCommands,
+		([signature, description], manualItem) ->
+			signature == manualItem or signature.startsWith("#{manualItem} ")
+	))
```

Comment:
The `print()` function is local to this module and all calls to it were manually tested. It was very tempting to convert the file to typescript, but there are other priorities...